README.md

```diff
@@ -283,6 +68,14 @@ We are eager to explore integrations with other computational backends such as [
 
 Have questions, suggestions, or comments? You can come hang out with us by joining the [CZI Science Slack](https://join-cellxgene-users.herokuapp.com/) and posting in the `#cellxgene-users` channel. As mentioned above, please submit any feature requests or bugs as [Github issues](https://github.com/chanzuckerberg/cellxgene/issues). We'd love to hear from you!
```

Comment:
Can we change the above section to communication? And maybe  developer guide -> contributing? I think that matches what user expectations are for those sections more closely.

Reply:
done.